website/versions.json

```diff
@@ -0,0 +1,3 @@
+[
+    "v0.1.x"
```

Comment:
This should be `v0.1.0` given how the release automation flow works: `semrel.outputs.version` is inferred based on whether the branch contains non-breaking feature changes, so the next time we run it, the `CLI_VERSION` will be either `v0.2.0` or `v0.1.1` and the `?.?.x` format will be a weird one-off.